ProblemsSolved.js

```diff
@@ -0,0 +1,76 @@
+function ProblemsSolved() 
+{
+}
+
+ProblemsSolved.prototype.SumOfEvenNumberInArray = function(array)
+{
+    var sum = 0;
+    for(i=0; i < array.length; i++)
```

Comment:
Asigning variables without declaration will define your variable in your global namespace (for browser -> window).
So when you say here for (i = 0...) will define i on window, so you will be able to access it through  window.i

If you were to write for (var i = 0; ... ), the "i" variable will be available only in your function's scope, and not reach the global context.